src/BuildScriptGeneratorCli/Commands/DockerfileCommand.cs

```diff
@@ -78,33 +79,49 @@ internal class DockerfileCommand : CommandBase
 
         internal override int Execute(IServiceProvider serviceProvider, IConsole console)
         {
-            ILoggerFactory loggerFactory = serviceProvider.GetRequiredService<ILoggerFactory>();
-            var sourceRepo = new LocalSourceRepo(this.SourceDir, loggerFactory);
-            var ctx = new DockerfileContext
+            var buildEventProps = new Dictionary<string, string>()
             {
-                SourceRepo = sourceRepo,
+                { "platform", this.PlatformName },
+                { "platformVersion", this.PlatformVersion },
+                { "runtimePlatformName", this.RuntimePlatformName },
```

Comment:
Some other inputs it could be useful to log are the `RuntimePlatformVersion` and `BuildImage`

Reply:
Agreed -- even though not all of these arguments will be provided to the `oryx dockerfile` call, it's still worth seeing what percentage of calls are being made with or without certain arguments, and when arguments are provided, what their value is. The other arguments don't need to be tracked in this case.

Reply:
alright. I can add these.